linux/deploy_vm/rebuild_unattend_install_iso.yml

```diff
@@ -21,245 +21,254 @@
 - name: "Set fact of source ISO image file name and path at local"
   ansible.builtin.set_fact:
     src_iso_file_name: "{{ datastore_file_download_result.dest | basename }}"
+    src_iso_file_dir: "{{ datastore_file_download_result.dest | dirname }}"
     src_iso_file_path: "{{ datastore_file_download_result.dest }}"
 
-# Get boot images from source ISO
-- include_tasks: get_boot_images_from_iso.yml
```

Comment:
If `get_boot_images_from_iso.yml` isn't needed any more, it can be removed.

Reply:
Done. Thanks